### PR TITLE
uboot-envtools: add support for gl-[ar150|domino|mifi]

### PR DESCRIPTION
**Pull request by kvuorine**
ar71xx: Change u-boot-env partition writable

Changed u-boot-env partition mount rw for:
gl-ar150
gl-domino
gl-mifi

uboot-envtools: Add ubootenv uci_config

Added ubootenv uci_config for:
gl-ar150
gl-domino
gl-mifi

Signed-off-by: Kimmo Vuorinen <kimmo.vuorinen@gmail.com>

### CODE REVIEW

File: package/boot/uboot-envtools/files/ar71xx [hunk 1]
```diff
@@ -101,6 +101,11 @@ wndr3700)
 wndr4300)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x40000" "0x20000"
 	;;
+gl-ar150|\
+gl-domino|\
+gl-mifi)
+	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x8000" "0x10000"
+	;;
```
Comment (adrianschmutzler) [inline package/boot/uboot-envtools/files/ar71xx]: Move up to ensure alphabetic sorting.

**Comment by adrianschmutzler** [comment]
Can you provide (and test) the same for gl-ar150 on ath79?

**Comment by kvuorine** [comment]
> Can you provide (and test) the same for gl-ar150 on ath79?

I can, probably have some time in january.

**Comment by kvuorine** [comment]
Somehow /etc/fw_env.config doesn't get generated on ath79 target and can't see why.

Manually creating that file using values provided will read and write ubootenv fine.

**Comment by kvuorine** [comment]
On target ath79 board naming had changed, fixed and tested working.

**Comment by adrianschmutzler** [comment]
Thanks. Please separate patches for ar71xx and ath79 into two separate commits.

**Comment by kvuorine** [comment]
I had hard time making up commit messages for these changes so three commits was what I came up with.

**Comment by adrianschmutzler** [comment]
Thanks, merged. I will backport to 19.07, too.

**Pull request closed without merging**